openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/WhatsNewModal.styles.less

```diff
@@ -37,27 +37,101 @@
   background: @white;
 }
 
+@keyframes scrollIn {
+  to {
+    bottom: 24px;
+  }
+}
+
 .whats-new-alert-container {
   position: fixed;
-  bottom: 24px;
+  bottom: -100%;
   right: 30px;
   z-index: 1;
+  background: @white;
+  animation: scrollIn 1s normal forwards ease-in-out;
+  animation-delay: 2s;
```

Comment:
i think we should use 1s, @chirag-madlani whats your thought? I am feeling its very delay.